src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtil.java

```diff
@@ -42,9 +42,9 @@ private ScopeUtil() {
     public static Scope getScopeFromMods(DetailAST aMods) {
         // default scope
         Scope returnValue = Scope.PACKAGE;
-        for (DetailAST token = aMods.getFirstChild(); token != null
-                && returnValue == Scope.PACKAGE;
-                token = token.getNextSibling()) {
+        for (DetailAST token = aMods.getFirstChild();
+                 token != null && returnValue == Scope.PACKAGE;
+                 token = token.getNextSibling()) {
```

Comment:
if this is formattign refactoring , please move to  separate commit.

Reply:
Reverted.